src/utils.ts

```diff
@@ -89,14 +89,21 @@ export async function updateVersionJson (file: string, context: VerifyReleaseCon
 
     context.logger.log(`Updating version in ${file}`);
     const content = await readFile(file, 'utf8');
-    const json = JSON.parse(content);
-    if (json.version === context.nextRelease.version) {
```

Comment:
I think I would find it better if `JSON.parse` and the early version check were left in. It was there anyway and it also checks whether the JSON is valid. Sure, it's a few milliseconds slower, but I don't think that's a big deal in this context. What do you think?